packages/core/src/Structures/Client.ts

```diff
@@ -66,6 +65,55 @@ export class Client extends EventEmitter {
         this.rest.setToken(this.options.token!);
     }
 
+    public setupPostgresProxy(clientId?: string): PgRemoteDatabase<typeof schema> {
+        return drizzle(async (sql, params, method) => {
+            try {
+                const queueName = `bot:query:${randomUUID()}`;
```

Comment:
You make new queue for every RPC? Not a problem but be sure to unconsume and it will be auto deleted by the auto delete feature